java/runtime/src/main/java/org/ray/runtime/AbstractRayRuntime.java

```diff
@@ -79,6 +79,22 @@ public AbstractRayRuntime(RayConfig rayConfig) {
     objectStoreProxy.put(objectId, obj, null);
   }
 
+
+  /**
+   * Store a serialized object in the object store.
+   *
+   * @param obj The serialized Java object to be stored.
+   * @return A RayObject instance that represents the in-store object.
+   */
+  public RayObject<Object> putSerialized(byte[] obj) {
```

Comment:
This method should be considered experimental for now. We may want to unify it with `ray.put` later and also make it consistent with the Python API.

Reply:
It might be slightly clearer to call it `putBuffer` or something like that.

Reply:
I see, I think I misunderstood the original purpose. `putSerialized` seems fine.